recipes/openconnect/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "openconnect" %}
+{% set version = "8.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # OpenConnect source
+  - url: ftp://ftp.infradead.org/pub/openconnect/openconnect-{{ version }}.tar.gz
+    sha256: 30e64c6eca4be47bbf1d61f53dc003c6621213738d4ea7a35e5cf1ac2de9bab1
+
+  # OpenConnect requires a configuration script to run.
+  # It is downloaded from a separate repo.
+  # See https://www.infradead.org/openconnect/vpnc-script.html
+  - git_url: git://git.infradead.org/users/dwmw2/vpnc-scripts.git
+    git_rev: c0122e891f7e033f35f047dad963702199d5cb9e
```

Comment:
Please build from a tarball, rather than a git repo

Reply:
I don't think there is any source of tarballs for that source.  I only need a single file from it.  Should I vendor that file directly into the recipe?

Reply:
Oh crap, I am totally wrong -- their website has a "snapshot" feature.  I'll fix this.